### PR TITLE
Minor protocol.txt improvements

### PR DESCRIPTION
**Pull request by AGSPhoenix**
Nothing really notable here, just minor cleanup of wording and spelling. Also spell out what path the configuration request should go to.

### CODE REVIEW

File: protocol.txt [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-    * cpu_cores: Number of core (or thread) available.
-    * ram: Memory available in kilo bytes.
-    * ram_max: Maximum memory allowed for render (in kilo bytes).
-    * extras (optional): Extra data use for the configuration.
-    * hostname (optional): Hostname of the machine, it's useful when the user have multiple machines with same hardware configuration. It's only used for display for server website.
+    * cpu_family: CPU's family, on linux it can be found in /proc/cpuinfo via the attribute "cpu family".
+    * cpu_model: CPU's model, on linux it can be found in /proc/cpuinfo via the attribute "model".
+    * cpu_model_name: CPU's model in human readable format, on linux it can be found in /proc/cpuinfo via the attribute "model name".
+    * cpu_cores: Number of cores (or thread) available.
+    * ram: Memory available in kilobytes.
```
Comment (MCOfficer) [inline protocol.txt]: put this `in kilobytes` in braces, too? just consistency

File: protocol.txt [hunk 2]
```diff
@@ -77,10 +77,10 @@ On error: an 404 http code
 
 Url: use the request type "request-job" from the configuration answer.
 Parameter as GET or POST:
-    * computemethod: What are my computer capability, 0 for using gpu and cpu (the computer is capable of using both but it will not use it at the same time), 1 for cpu only, 2 for gpu only. 
+    * computemethod: What compute types are available on this machine. 0 for CPU or GPU, 1 for CPU only, 2 for GPU only. 
     * cpu_cores: Number of cores currently available for rendering (optional).
     * gpu_model: Model name of the GPU available for rendering
-    * gpu_ram: GPU's memory (in byte)
+    * gpu_ram: GPU memory size (in bytes)
```
Comment (MCOfficer) [inline protocol.txt]: is `size` really necessary?

File: protocol.txt [hunk 3]
```diff
@@ -77,10 +77,10 @@ On error: an 404 http code
 
 Url: use the request type "request-job" from the configuration answer.
 Parameter as GET or POST:
-    * computemethod: What are my computer capability, 0 for using gpu and cpu (the computer is capable of using both but it will not use it at the same time), 1 for cpu only, 2 for gpu only. 
+    * computemethod: What compute types are available on this machine. 0 for CPU or GPU, 1 for CPU only, 2 for GPU only.
```
Comment (MCOfficer) [inline protocol.txt]: `0 for CPU and GPU` or `0 for both CPU and GPU`

File: protocol.txt (hunk 2 above)
Comment (AGSPhoenix) [reply protocol.txt]: Probably not, but just 'GPU memory' seemed a bit vague. I have no strong feelings either way.

File: protocol.txt (hunk 3 above)
Comment (AGSPhoenix) [reply protocol.txt]: The intent with this was to remove the clunky explanation that each job will only use CPU or GPU, hence the switch from 'and' to 'or'. I think it's clear as proposed, especially give the descriptions on the other two options.

Comment (MCOfficer) [reply protocol.txt]: fair point, i don't expect anyone unexperienced messing with this anyways

File: protocol.txt (hunk 2 above)
Comment (MCOfficer) [reply protocol.txt]: me neither, just my 2 cents... as i said, nitpicking ;)

**Pull request merged**